xml/decoders/TCS_WOWDieselV5-121.xml

```diff
@@ -0,0 +1,12737 @@
+            <enumChoice choice="Gyra Light" value="7"/>
+            <enumChoice choice="Mars Light" value="2"/>
+            <enumChoice choice="Rotary Beacon (New Style)" value="69"/>
+            <enumChoice choice="Rotary Beacon (Old style)" value="6"/>
+            <enumChoice choice="Rule 17" value="8"/>
+            <enumChoice choice="Single Pulse Strobe" value="4"/>
+          </enumChoiceGroup>
+        </enumVal>
+        <label>              F0R (Yellow) Effect</label>
+      </variable>
+      <variable default="0" item="Output 3 effect generated" mask="VVXXVVVV" CV="53" minOut="3">
+        <value>0</value>
+        <enumVal>
+          <enumChoiceGroup xsi:noNamespaceSchemaLocation="http://jmri.org/xml/schema/decoder-4-15-2.xsd">
+            <enumChoice choice="Auto Mars" value="13"/>
+            <enumChoice choice="Brake Lights" value="14"/>
+            <enumChoice choice="Constant Bright" value="0"/>
+            <enumChoice choice="Constant Dim" value="12"/>
+            <enumChoice choice="Custom Light 1" value="70"/>
+            <enumChoice choice="Custom Light 2" value="71"/>
+            <enumChoice choice="Ditch Light Side 1" value="10"/>
+            <enumChoice choice="Ditch Light Side 2" value="11"/>
+            <enumChoice choice="Double Pulse Strobe" value="5"/>
+            <enumChoice choice="Firebox Flicker" value="1"/>
+            <enumChoice choice="Flashing F.R.E.D." value="3"/>
+            <enumChoice choice="Gyra Light" value="7"/>
+            <enumChoice choice="Mars Light" value="2"/>
+            <enumChoice choice="Rotary Beacon (New Style)" value="69"/>
+            <enumChoice choice="Rotary Beacon (Old style)" value="6"/>
+            <enumChoice choice="Rule 17" value="8"/>
+            <enumChoice choice="Single Pulse Strobe" value="4"/>
+          </enumChoiceGroup>
+        </enumVal>
+        <label>F3 (Brown) Effect</label>
+      </variable>
+      <variable default="0" item="Output 3 behavior" mask="XXVVXXXX" CV="53" minOut="3">
+        <enumVal>
+          <enumChoice choice="Function on running forward" value="0"/>
+          <enumChoice choice="Function on running reverse" value="1"/>
+          <enumChoice choice="Function on both directions" value="2"/>
+        </enumVal>
+        <label>F3 (Brown) Directionality</label>
+      </variable>
+      <variable default="0" item="Function 2 effect generated" mask="VVXXVVVV" CV="52" minOut="2">
+        <value>0</value>
+        <enumVal>
+          <enumChoiceGroup xsi:noNamespaceSchemaLocation="http://jmri.org/xml/schema/decoder-4-15-2.xsd">
+            <enumChoice choice="Auto Mars" value="13"/>
+            <enumChoice choice="Brake Lights" value="14"/>
+            <enumChoice choice="Constant Bright" value="0"/>
+            <enumChoice choice="Constant Dim" value="12"/>
+            <enumChoice choice="Custom Light 1" value="70"/>
+            <enumChoice choice="Custom Light 2" va
```

Comment:
Lots of other validation errors:
![Screen Shot 2022-01-14 at 3 15 52 pm](https://user-images.githubusercontent.com/13841457/149455665-08ce20b4-df07-44aa-8f2e-1141a812560d.png)

Reply:
@dheap Thanks for the points here. The tricky part with the validator has been: with certain elements and attributes if you match the validator it fails in operation and vice versa. I found this when I initially started getting beta test feedback on decoder definitions and while I am extremely open to needing to clean things up or sharpen them some, those conflicts drastically reduced my trust in the validator (mask and splitVal on indexed CV's was a particular case. 

To sanity check myself when I first discovered this I ran some other (currently adopted) manufacturer definitions through the validator and found quite a number of validation failures as well, though the definitions work well as intended, so I'm sure you can see where the trust of that particular tool is a little shakey.

I'm triaging and strategizing with leadership at TCS to see how to proceed next, but from the top here the read back bug is the only hang-up/concern they have.